frontends/web/src/locales/en/app.json

```diff
@@ -68,13 +68,11 @@
       "step": "Finished",
       "title": "Account added"
     },
-    "title": "Add account"
+    "title": "+ Add account"
   },
   "aopp": {
     "addressRequest": "{{host}} is requesting a receiving address.",
-    "addressRequestWithLogo": "is requesting a receiving address.",
-    "addressRequestWithLogoAndXPub": "is requesting a receiving address and public key (xpub).",
-    "addressRequestWithXPub": "{{host}} is requesting a receiving address and public key (xpub).",
```

Comment:
These keys are new and should not be removed. Did you push?

https://github.com/BitBoxSwiss/bitbox-wallet-app/commit/5220df6948aff065fb7bc1f75f4c4c57e45c37d5#diff-6631e5e7b36d6349336cdf14e4f91eeeb8f284d449b8281af375936bbe94ae79

Reply:
I did, but after pull :see_no_evil: fixed now